handlers/sql_generic_v3/readme.md

```diff
@@ -0,0 +1,81 @@
+# Sql Generic V3
+
+The generic sql handler uses a predefined query to execute against a SQL database.  The predefined queries are kept in a Space Form submission and retrieved when the handler runs.  The Space Form's slug is defined as a info value for the handler.  
+In order for the handler to retrieve the correct query the Space Form needs a unique index to search on.  The handler expects the Space From to have a field with the name **Template Name** and the form should use that field as a unique index.
+
+## Info values
+[database_server]
+
+	JDBC connection that specifies the database location.
+[database_port]
+
+	The port the database is on
+[database_username]
+
+	The username used to connect to the database
+[database_password]
+
+	The password for the user used to connect to the database
+[kinetic_api_location]
+
+	API Location (https://acme.kinops.io/app/api/v1).
+[kinetic_api_username]
+
+	The username for impersonated user.
+[kinetic_api_password]
+
+	The impersonated user's password.
+[space_form]
+
+	The form slug for the Space Form where the query template is saved.
+[enable_debug_logging]
+	
+	Enable debug logging if the value is set to 'Yes'.
+
+## Parameters
+
+[Error Handling]
+
+	Determine what to return if an error is encountered.
+[Database]
+
+	Database to Access for example sqlserver, oracle or postgresql.
+[Database Name]
+
+	Database Name

```

Comment:
I don't understand why these are in Parameters and not Info Values. Especially Database, because that is tied to all the other properties that are currently in Info Values.